tests/frameworks/pytorch/hebbian_pruning_test.py

```diff
@@ -0,0 +1,322 @@
+#  Numenta Platform for Intelligent Computing (NuPIC)
+#  Copyright (C) 2019, Numenta, Inc.  Unless you have an agreement
```

Comment:
Can you move this to `nupic.research/tests/frameworks/dynamic_sparse`?

Reply:
Done